src/main/kotlin/com/devfloor/untitled/article/presentation/ArticleController.kt

```diff
@@ -0,0 +1,19 @@
+package com.devfloor.untitled.article.presentation
+
+import com.devfloor.untitled.article.application.ArticleFacade
+import com.devfloor.untitled.article.application.ArticleResponse
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ArticleController(
+    private val articleFacade: ArticleFacade
+) {
+    @GetMapping("/articles/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    fun show(@PathVariable id: Long): ArticleResponse =
+        articleFacade.showByArticleId(id)
```

Comment:
여기도 생각해보니 표현식이니까 반환 타입 제거해도 될 듯

```suggestion
    fun show(@PathVariable id: Long) = articleFacade.showByArticleId(id)
```